lib/dynamodb_mutex/lock.rb

```diff
@@ -1,4 +1,5 @@
-require 'aws-sdk-v1'
+require 'aws-sdk'
+require 'socket'
```

Comment:
It is needed for `Socket.gethostname` in the `pid` method. I think `aws-sdk-v1` loaded it previously, as the `require` became necessary when `aws-sdk-v1` was replaced with `aws-sdk`.